components/docs-chef-io/content/automate/ha_upgrade_introduction.md

```diff
@@ -79,4 +79,105 @@ We can also pass a flag in upgade command to avoid prompt for workspace upgrade.
    chef-automate upgrade run --airgap-bundle latest.aib --auto-approve --workspace-upgrade no
   ```
 
+{{< note >}}
+
+  AMI Upgrade is only for AWS deployment ,as in Onpremise Deployment all the resources are managed by customers its self.  
+{{< /note >}}
+
+## AMI Upgrade Setup For AWS Deployment
+
+### Requirements
+
+1. Two identical clusters located in same/different AWS regions.
+1. Amazon S3 access in both regions for Application backup.
+
+In the above approach, there will be two identical clusters
+
+- Primary Cluster (or Production Cluster)
+- Secondary Cluster (Having upgraded AMI)
+
+{{< note >}}
+
+  The AWS deployment should be configured with S3.  
```

Comment:
Add more details to this note. 
1. Which S3 bucket should be configured to the new (upgraded) cluster setup
2. Now with [MADROX-427](https://chefio.atlassian.net/browse/MADROX-427), user can directly mention existing bucket name in config.toml

Reply:
Done